src/sage/categories/functor.pyx

```diff
@@ -477,7 +487,7 @@ class ForgetfulFunctor_generic(Functor):
         """
         TESTS::
 
-            sage: F = ForgetfulFunctor(FiniteFields(),Fields())
+            sage: F = ForgetfulFunctor(FiniteFields(), Fields())
             sage: F #indirect doctest
             The forgetful functor from Category of finite enumerated fields to Category of fields
```

Comment:
want to split this output line?

Reply:
In TESTS blocks, I have been less strict with the formatting because by default it is not shown in the formatted documentation.
Same for docstrings of private/special methods unless they appear in the formatted documentation